R/dl_build_src_gran.R

```diff
@@ -72,6 +82,20 @@ dl_build_src <- function(GRAN.dir = './GRAN'){
 		cat('Installing missed packages:', missed_pkgs)
 		install.packages(unique(missed_pkgs), type='source', repos=paste0('file:', GRAN.dir))
 	}
+	
+	#write updated build list
+	oldList <- read.table(currentListPath, sep='\t', header=TRUE,stringsAsFactors=FALSE)
+	for(i in 1:nrow(packages)){ #could be vectorized?
+	  oldList[grepl(packages$package[i],oldList$Package),2] <- packages$tag[i]
+	}
+	updatedList <- oldList
+	write.table(updatedList,paste0(src_dir,"/updatedBuildTags_",version,".tsv"), quote = FALSE, row.names = FALSE)
```

Comment:
drop version from here. Also, this file path should be used for comparison on future runs, right? not `./granCurrent.tsv`?